src/components/CarouselCard/__tests__/indicators.spec.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Indicators from '../indicators';
+
+const carouselChildren = [
+    {
+        indicatorID: 'indicator-1',
+        containerID: 'container-1',
+        ref: {
+            click: jest.fn(),
+        },
+        header: 'Header',
+    },
+];
+
+describe('<Indicators />', () => {
```

Comment:
add test for:
* should set the role as tablist in ul element
* should set the role as tab in anchor element
* should set the tabIndex as -1 and aria-selected to false in anchor element when is not selected
* should set title and text in AssistiveText to undefined when header is not a string